### DISCUSSION TITLE
Is it possible to monitor a host from a group of pollers ?

### DISCUSSION THREAD

**Discussion by chrisdef** [centreon-archive/centreon-archived#9496] [Q&A] [answered]
Hi,

I'm looking if it possible to configure a host to be monitored from a group of pollers instead of one unique poller ?

I didn't find anything about failover for Pollers in the documentation.

Please let me know if it is possible?

Best regards.
Chris

**Comment by cgagnaire** [comment, marked as answer]
Hi,
It is not possible for now, but it is more or less on our roadmap.

**Reply by chrisdef** [reply]
Hi,
Thanks for the answer. Hope to hear about it soon 👍
